python/cugraph/tests/test_random_walks.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.:
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+
+import pytest
+
+from cugraph.tests import utils
+import cugraph
+import random
+
+
+# =============================================================================
+# Parameters
+# =============================================================================
+DIRECTED_GRAPH_OPTIONS = [False, True]
+WEIGHTED_GRAPH_OPTIONS = [False, True]
+DATASETS = [pytest.param(d) for d in utils.DATASETS]
```

Comment:
I've been adding `ids` to make it easier to see what dataset is being run in the event of a failure:
```suggestion
DATASETS = [pytest.param(d) for d in utils.DATASETS,
            ids=[f"dataset={d.as_posix()}" for d in utils.DATASETS]]
```